app/migrations/Version20170429092049.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Mautic\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20170429092049 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        $columns = $schema->getTable($this->prefix.'emails')->getColumns();
+
+        if (array_key_exists('utm_tags', $columns)) {
```

Comment:
The two lines above could be written more easily with
`if ($schema->getTable($this->prefix.'emails')->hasColumn('utm_tags')) {`
It does the same thing, but the code is more easily readable.